workflow-templates/im-run-tf-destroy.yml

```diff
@@ -11,11 +11,15 @@
 # TODO Prerequisites
```

Comment:
This workflow is meant to be a very lightweight runbook for destroying a resource, why do we need to expand it to do the stakeholder/attestor approvals and do a terraform apply and update a deployment board?

Reply:
I added this so you can approve what you are destroying.  My team suggested stake holder approval, but I can remove that and just have Terraform approval.  The deployment board gives status of the destroy.  The bare bones one I used and was lacking with approvals.  It just destroys stuff.  No step to validate is very dangerous if you select the wrong environment.

Reply:
@danielle-casella-adams  would you prefer I just create a second workflow call `im-run-tf-destroy-approvals.yml` so we have both.